### PR TITLE
[Tooling] Maintenance terminal shortcut

### PR DESCRIPTION
**Pull request by petertgiles**
## 👋 Introduction

Here's a little branch to add a terminal shortcut for the maintenance container in VS Code.  I've found it useful, you might too.

Feel free to add to this branch with extra settings if you're a OSX or Win person.

## 📸 Screenshot

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -77,5 +77,33 @@
...
   "laravel-pint.configPath": "api/pint.json",
-  "laravel-pint.executablePath": "api/vendor/bin/pint"
+  "laravel-pint.executablePath": "api/vendor/bin/pint",
+  "terminal.integrated.profiles.linux": {
+    "Maintenance": {
+      "path": "docker-compose",
+      "args": [
+         "run",
+         "--rm",
+         "--workdir=/var/www/html"
```
Comment (mnigh) [inline .vscode/settings.json:87]:
```suggestion
         "--workdir=/var/www/html",
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -77,5 +77,33 @@
...
+         "maintenance"
```
Comment (mnigh) [inline .vscode/settings.json:88]:
```suggestion
         "maintenance",
```

File: .vscode/settings.json [hunk 3]
```diff
@@ -77,5 +77,33 @@
...
+         "bash"
```
Comment (mnigh) [inline .vscode/settings.json:89]:
```suggestion
         "bash",
```

**Review by esizer — APPROVED** [review]
Works! Amazing, thank-you 🙇‍♀️

**Pull request merged**